### PR TITLE
[backport] Windows: update wget and remove --quiet from wget command 

### PR DESCRIPTION
**Pull request by AlwinEsch**
## Description

Fix binary addons build on Windows. Currently does by all addons on Matrix Jenkins fails with Windows builds.

---------------------------------------------

**Backport about https://github.com/xbmc/xbmc/pull/19588 with text:**

It seems to be non stop problems with windows builds these days so this is my attempt to solve them.

I've updated wget to the only version I could find (no idea if this source is trustworthy or not wink).

I've also removed the --quiet switch so we can actually see what happened if a download failed.

I've tested running the download-dependencies.bat on the Azure builder a few times with success.

@phunkyfish can you see if this solves the download issues you were having when doing windows builds?

## Types of change

- [ ] **Bug fix** (non-breaking change which fixes an issue)
- [ ] **Clean up** (non-breaking change which removes non-working, unmaintained functionality)
- [ ] **Improvement** (non-breaking change which improves existing functionality)
- [ ] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that will cause existing functionality to change)
- [ ] **Cosmetic change** (non-breaking change that doesn't touch code)
- [ ] **None of the above** (please explain below)

## Checklist:

- [ ] My code follows the **[Code Guidelines](https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md)** of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [ ] I have read the **[Contributing](https://github.com/xbmc/xbmc/blob/master/docs/CONTRIBUTING.md)** document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

### CODE REVIEW

**Review by lrusak — APPROVED** [review]
Thanks for doing this. I'm not sure if we want to wait for @DaveTBlake 's approval but it's good to go from me.

**Review by wsnipex — APPROVED** [review]

**Review by DaveTBlake — APPROVED** [review]
Thanks for the ping, but yes let's get this in so we can see what is happening

**Pull request merged**
